community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -409,6 +409,11 @@
     public static final Setting<Boolean> multi_threaded_schema_index_population_enabled =
             setting( "unsupported.dbms.multi_threaded_schema_index_population_enabled", BOOLEAN, TRUE );
 
+    // Bloom index
+    @Internal
+    public static final Setting<List<String>> bloom_indexed_properties =
```

Comment:
we should not expose extension settings in global database settings file, please introduce plugin local setting file instead